invenio_celery/ext.py

```diff
@@ -26,24 +26,39 @@
 
 from __future__ import absolute_import, print_function
 
+import pkg_resources
 import time
 
 from flask_celeryext import FlaskCeleryExt
 
 
 class InvenioCelery(object):
-    """Invenio theme extension."""
+    """Invenio celery extension."""
 
     def __init__(self, app=None, **kwargs):
         """Extension initialization."""
         self.celery = None
+
         if app:
             self.init_app(app, **kwargs)
 
-    def init_app(self, app, assets=None, **kwargs):
+    def init_app(self, app, assets=None,
+                 entrypoint_name="invenio_celery.tasks", **kwargs):
         """Initialize application object."""
         self.init_config(app.config)
         self.celery = FlaskCeleryExt(app).celery
+
+        # this reduces signature complexity, we want False only for testing
```

Comment:
```python
if entrypoint_name:
    all_modules = []
    for item in pkg_resources.iter_entry_points(group=entrypoint_name):
        all_modules.append(item.module_name)
    self.celery.autodiscover_tasks(all_modules, related_name="", force=True)
```

If you want to disable entrypoint loading you simply use ``InvenioCelery(entrypoint_name=None)``. I don't think entry point loading without auto discovery makes sense.